deploifai/project/init.py

```diff
@@ -0,0 +1,90 @@
+import click
+import click_spinner
+from PyInquirer import prompt
+from click import Abort
+import os
+
+from deploifai.api import DeploifaiAPIError
+from deploifai.context import (
+    pass_deploifai_context_obj,
+    DeploifaiContextObj,
+    is_authenticated,
+)
+from deploifai.utilities.config import local_config
+
+
+@click.command()
+@pass_deploifai_context_obj
+@is_authenticated
+def init(context: DeploifaiContextObj):
+    """
+    Store project information on your local device
```

Comment:
should be `Initialize current working directory as a Deploifai project`. I think we're gonna follow the convention we have for dataset init as well, to just initialize the current working directory as the project directory

Reply:
okay